### PR TITLE
Fix env variables in documentation

### PR DESCRIPTION
**Pull request by friedjoff**

### CODE REVIEW

File: src/components/TrafimageMaps/TrafimageMaps.js [hunk 1]
```diff
@@ -203,10 +203,10 @@ const defaultProps = {
...
   menus: null,
   subMenus: null,
-  cartaroUrl: null,
-  geoServerUrl: null,
-  vectorTilesKey: null,
-  vectorTilesUrl: null,
+  cartaroUrl: process.env.REACT_APP_CARTARO_URL,
+  geoServerUrl: process.env.REACT_APP_GEOSERVER_URL,
+  vectorTilesKey: process.env.REACT_APP_VECTOR_TILES_KEY,
+  vectorTilesUrl: process.env.REACT_APP_VECTOR_TILES_URL,
```
Comment (oterral) [inline src/components/TrafimageMaps/TrafimageMaps.js]: Can't we put prod urls by default? Don't forget  the app and the web-compoent will be separate. So these variables will not exist.

Comment (oterral) [reply src/components/TrafimageMaps/TrafimageMaps.js]: or may be they will also exits in the web component build.

File: src/AppRouter.js [hunk 2]
```diff
@@ -48,10 +48,6 @@ const AppRouter = () => (
             elements={elements}
             initialState={{ ...match.params }}
             popupComponents={POPUP_CONF}
-            cartaroUrl={process.env.REACT_APP_CARTARO_URL}
-            geoServerUrl={process.env.REACT_APP_GEOSERVER_URL}
-            vectorTilesKey={process.env.REACT_APP_VECTOR_TILES_KEY}
-            vectorTilesUrl={process.env.REACT_APP_VECTOR_TILES_URL}
```
Comment (oterral) [inline src/AppRouter.js]: We will still need this Ithink when the AppRouter will exits only in gitlab

Comment (friedjoff) [reply src/AppRouter.js]: Yeah, we can add them again, if needed. Just wanted to move the default value into TrafimageMaps.

File: src/components/TrafimageMaps/TrafimageMaps.js (hunk 1 above)
Comment (friedjoff) [reply src/components/TrafimageMaps/TrafimageMaps.js]: THE web component would need to support the .env file. In that case it's the same as putting prod urls by default in here.

Comment (oterral) [reply src/components/TrafimageMaps/TrafimageMaps.js]: yes

**Pull request merged**